ansible/files/telemetry.sh

```diff
@@ -1,140 +1,92 @@
-sudo yum -y install git jq htop tmux libffi-devel aws-cli postgresql-devel zsh snappy-devel readline-devel
+TELEMETRY_CONF_BUCKET=s3://telemetry-spark-emr-2
+MEMORY_OVERHEAD=7000  # Tuned for c3.4xlarge
+EXECUTOR_MEMORY=15000M
+DRIVER_MEMORY=$EXECUTOR_MEMORY
+
+# Install packages
+curl https://bintray.com/sbt/rpm/rpm | sudo tee /etc/yum.repos.d/bintray-sbt-rpm.repo
+sudo yum -y install git jq htop tmux libffi-devel aws-cli postgresql-devel zsh snappy-devel readline-devel sbt
 
-# Install external packages, e.g. emacs
+# Install custom packages, e.g. emacs
 mkdir packages
-aws s3 cp --recursive s3://telemetry-spark-emr/packages/ ./packages
+aws s3 cp --recursive $TELEMETRY_CONF_BUCKET/packages/ ./packages
 sudo yum -y install packages/*
 
-INSTANCES=$(jq .instanceCount /mnt/var/lib/info/job-flow.json)
-FLOWID=$(jq -r .jobFlowId /mnt/var/lib/info/job-flow.json)
-EXECUTORS=$(($INSTANCES>1?$INSTANCES:2 - 1))
-EXECUTOR_CORES=$(nproc)
-MAX_YARN_MEMORY=$(grep /home/hadoop/conf/yarn-site.xml -e "yarn\.scheduler\.maximum-allocation-mb" | sed 's/.*<value>\(.*\).*<\/value>.*/\1/g')
-EXECUTOR_MEMORY=$(echo "$MAX_YARN_MEMORY * 0.65" | bc | cut -d'.' -f1)
-MEMORY_OVERHEAD=$(echo "$MAX_YARN_MEMORY - 1024 - $EXECUTOR_MEMORY" | bc | cut -d'.' -f1)
-EXECUTOR_MEMORY=${EXECUTOR_MEMORY}M
-DRIVER_MEMORY=$EXECUTOR_MEMORY
-HOME=/home/hadoop
-
-# Error message
-error_msg ()
-{
-	echo 1>&2 "Error: $1"
-}
+# Download jars
+aws s3 sync $TELEMETRY_CONF_BUCKET/jars $HOME/jars
```

Comment:
You removed the explicit setting of the `HOME` var - is this the desired behaviour?